### DISCUSSION TITLE
Configurable background color for screen shots.

### DISCUSSION THREAD

**Discussion by harlan3** [hizzgdev/jsmind#574] [General]
Screen shots are currently saved as a transparent background that shows up as black in many viewers. It would be good to have the background color configurable so that the screen shots can be printed more easily by using a lighter background color.

**Comment by hizzgdev** [comment]
Thanks, I agree that. Will try to upgrade next month. And It will be appreciated if anyone can submit a PR for that !

**Reply by hizzgdev** [reply]
https://github.com/hizzgdev/jsmind/pull/576
